.github/ISSUE_TEMPLATE/0-bug-report.yml

```diff
@@ -0,0 +1,56 @@
+name: "🐛 Bug Report"
+description: Create a report to help us improve
+labels: ['bug']
+
+body:
+  - type: markdown
+    attributes:
+      value: |
+        **Never report security issues on GitHub or other public channels (Gitter/Twitter/etc.).** Follow the instruction from [Jenkins Security](https://jenkins.io/security/) or use our [TraceTronic Support Center](https://support.tracetronic.com).
+        For reporting issues containing NDA relevant information please also use our [TraceTronic Support Center](https://support.tracetronic.com).
+  - type: textarea
+    id: description
+    attributes:
+      label: Describe the bug
+      description: A clear and concise description of what the bug is.
+    validations:
+      required: true
+  - type: textarea
+    id: reproduce
+    attributes:
+      label: Steps to reproduce
+      description: List of steps to reproduce the behavior.
+    validations:
+      required: true
+  - type: textarea
+    id: expectation
+    attributes:
+      label: Expected behavior
+      description: A clear and concise description of what you expected to happen.
+    validations:
+      required: true
+  - type: input
+    id: jeninks-version
+    attributes:
+      label: Jenkins version
+    validations:
+      required: true
+  - type: input
+    id: plugin-version
+    attributes:
+      label: ECU-TEST-Execution Plugin version
+    validations:
+      required: true
+  - type: input
+    id: et-version
+    attributes:
+      label: ECU-TEST version
+  - type: input
+    id: tg-version
+    attributes:
+      label: TEST-GUIDE version
```

Comment:
The bug issue result looks not so good. Maybe its better to use
- type: textarea
    attributes:
      label: Versions
      description: |
        examples:
          - **Jenkins**: LTS 2.332.3
          - **Plugin**:  3.0
          - **ECU-TEST**: 2022.2
          - **TEST-GUIDE**:  1.139.0 (or empty)
      value: |
          - Jenkins:
          - Plugin:
          - ECU-TEST:
          - TEST-GUIDE:
      render: markdown
    validations:
      required: true